resources/windows/wix/service.component.wxs.erb

```diff
@@ -0,0 +1,58 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+  <%- get_services.each do |service| -%>
+    <%- service.options.each do |component| -%>
+    <DirectoryRef Id="<%= component[:directory_ref] %>">
+      <Component Id='<%= component[:id] %>' Guid="<%= component[:guid] %>" <%= "Win64=\"#{component[:win64]}\"" unless component[:win64].nil? %>>
+        <%- if component[:file][:id] == "NSSM" -%>
+        <CreateFolder />
```

Comment:
@jcoconnor are you telling me this isn't a thing we need in this file? What are you saying with that comment?